src/graph_notebook/notebooks/04-Machine-Learning/Neptune-ML-06-Text-Encoding-Tutorial.ipynb

```diff
@@ -0,0 +1,1281 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "s3_bucket_uri=\"s3://<INSERT S3 BUCKET OR PATH>\"\n",
+    "s3_bucket_uri = s3_bucket_uri[:-1] if s3_bucket_uri.endswith('/') else s3_bucket_uri"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "export_params={ \n",
+    "\"command\": \"export-pg\", \n",
+    "\"params\": { \"endpoint\": neptune_ml.get_host(),\n",
+    "            \"profile\": \"neptune_ml\",\n",
+    "            \"cloneCluster\": False\n",
+    "            }, \n",
+    "\"outputS3Path\": f'{s3_bucket_uri}/neptune-export',\n",
+    "\"additionalParams\": {\n",
+    "        \"neptune_ml\": {\n",
+    "        \"version\": \"v2.0\",\n",
+    "        \"targets\": [\n",
+    "            {\n",
+    "            \"edge\": [\"user\", \"apply\", \"job\"],\n",
+    "            \"type\" : \"link_prediction\",\n",
+    "            \"split_rate\": [0.8,0.1,0.1]\n",
+    "            }\n",
+    "        ],\n",
+    "         \"features\": [\n",
+    "            {\n",
+    "                \"node\": \"user\",\n",
+    "                \"property\": \"Major\",\n",
+    "                \"type\": \"text_sbert\"\n",
+    "            },\n",
+    "            {\n",
+    "                \"node\": \"job\",\n",
+    "                \"property\": \"Title\",\n",
+    "                \"type\": \"text_sbert\",\n",
+    "            },\n",
+    "            {\n",
+    "                \"node\": \"job\",\n",
+    "                \"property\": \"Description\",\n",
+    "                \"type\": \"text_fasttext\",\n",
+    "                \"language\": \"en\"\n",
+    "            },\n",
+    "            {\n",
+    "                \"node\": \"job\",\n",
+    "                \"property\": \"Requirements\",\n",
+    "                \"type\": \"text_fasttext\",\n",
+    "                \"language\": \"en\"\n",
+    "            }\n",
+    "         ]\n",
+    "        }\n",
+    "      },\n",
+    "\"jobSize\": \"medium\"}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "b41a3c624a8e4dd9bf1f5b76e16441a8",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Output()"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "%%neptune_ml export start --export-url {neptune_ml.get_export_service_host()} --export-iam --wait --store-to export_results\n",
```

Comment:
Will this cell show that export completes or how can a customer check before moving on to step 3?

Reply:
Yes. Only the export has been finished can we move on to the next step/cell. If this step is done. We will also show the message.

Reply:
We should revise this section. It goes through a lot of background and then presents 2 commands without context. We should add a prefacing sentence before each cell to explain what running the cell does and what results can be expected.

Reply:
fixed by adding 'The indication of `Export is finished` along with `"status": "succeeded"` shows the success of the export job.'